### DISCUSSION TITLE
dbeaver-ce-23.3.4-x86_64-setup.exe packages Version 23.3.2!

### DISCUSSION THREAD

**Discussion by treysis** [dbeaver/dbeaver#22746] [Q&A]
### Description

Download the windows installer for 23.3.4 https://github.com/dbeaver/dbeaver/releases/download/23.3.4/dbeaver-ce-23.3.4-x86_64-setup.exe
After installation, the main window and the "about" dialog both show version 23.3.**2**.

### DBeaver Version

23.3.2 or 23.3.4

### Operating System

Windows 10 22H2

### Steps to reproduce

Download the windows installer for 23.3.4 https://github.com/dbeaver/dbeaver/releases/download/23.3.4/dbeaver-ce-23.3.4-x86_64-setup.exe
After installation, the main window and the "about" dialog both show version 23.3.**2**.

**Comment by E1izabeth** [comment]
When you run the installer, there is a choice of the installation way and place
![image](https://github.com/dbeaver/dbeaver/assets/28875055/21233110-2618-49ed-ab13-7e7dda596d49)

Is it possible that you chose an option other than the one selected during the initial installation? I suspect that there are 2 applications installed on your computer now - one installed for all users and another one installed for current user.

Could you check? To check it, you can try changing this switcher in the update wizard.

**Comment by treysis** [comment]
I only ever selected "For me" as a user-only install. The other option wouldn't even work as I don't have admin privileges on this computer.

However, I got mixed up with the parallel install of the Microsoft Store version of DBeaver which is still at 23.3.2. My bad 😅
